OloEditor/src/Panels/ContentBrowserPanel.cpp

```
@@ -6,120 +6,120 @@
-				return icon;
-			}
-			else
-			{
-				OLO_WARN("Could not load texture {0}", extension);
-			}
-		}
-		return m_FileIcon;
-	}
-
-}
+    ContentBrowserPanel::ContentBrowserPanel()
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_BaseDirectory, m_CurrentDirectory, m_DirectoryIcon, m_FileIcon

(cppcoreguidelines-pro-type-member-init)